packages/web/src/components/modal-radio/ModalRadioItem.module.css

```diff
@@ -20,10 +20,11 @@
   display: flex;
   align-items: center;
   gap: var(--unit-2);
-  margin-left: var(--unit-2);
-  font-size: var(--font-2xl);
-  font-weight: var(--font-bold);
-  line-height: 150%;
+}
+
+.input {
+  height: 24px;
```

Comment:
var(--unit-6) cause why not nitpick

Reply:
i thought the unit vars were only for spacing, but don't to use everywhere if that makes more sense to folks

Reply:
Yeah I think you're right that for height and width it's more of a tossup, long term I'd like to see some consistency here since the values should interplay nicely with spacing ya know?